### ISSUE TITLE
Error while trying to bin data

### ISSUE THREAD

**Issue by achanda** [metabase/metabase#5929]
Browser: Chrome Version 60.0.3112.113 (Official Build) (64-bit)
OS: OSX 10.12.6
DB: postgres
Metabase version: v0.25.2
Hosting env: in a docker container talking to another container running postgres using docker links
Metabase internal db: H2

I am trying to run a question which bins data in the sample db, this fails with the error `Illegal clause (no matching fn found): :binning-strategy`. The container logs show
```
09-11 14:06:32 WARN metabase.query-processor :: Query failure: Illegal clause (no matching fn found): :binning-strategy
["query_processor$assert_query_status_successful.invokeStatic(query_processor.clj:201)"
 "query_processor$assert_query_status_successful.invoke(query_processor.clj:194)"
 "query_processor$run_and_save_query_BANG_.invokeStatic(query_processor.clj:233)"
 "query_processor$run_and_save_query_BANG_.invoke(query_processor.clj:227)"
 "query_processor$fn__27764$process_query_and_save_execution_BANG___27769$fn__27770.invoke(query_processor.clj:270)"
 "query_processor$fn__27764$process_query_and_save_execution_BANG___27769.invoke(query_processor.clj:256)"
 "api.dataset$fn__29566$fn__29569.invoke(dataset.clj:63)"
 "api.common.internal$do_with_caught_api_exceptions.invokeStatic(internal.clj:227)"
 "api.common.internal$do_with_caught_api_exceptions.invoke(internal.clj:222)"
 "api.dataset$fn__29566.invokeStatic(dataset.clj:54)"
 "api.dataset$fn__29566.invoke(dataset.clj:54)"
 "middleware$enforce_authentication$fn__28955.invoke(middleware.clj:122)"
 "api.routes$fn__40141.invokeStatic(routes.clj:60)"
 "api.routes$fn__40141.invoke(routes.clj:60)"
 "routes$fn__40784$fn__40785.doInvoke(routes.clj:75)"
 "routes$fn__40784.invokeStatic(routes.clj:71)"
 "routes$fn__40784.invoke(routes.clj:71)"
 "middleware$log_api_call$fn__29054$fn__29056.invoke(middleware.clj:331)"
 "middleware$log_api_call$fn__29054.invoke(middleware.clj:330)"
 "middleware$add_security_headers$fn__29004.invoke(middleware.clj:245)"
 "middleware$bind_current_user$fn__28959.invoke(middleware.clj:142)"
 "middleware$maybe_set_site_url$fn__29008.invoke(middleware.clj:268)"]
```
Screenshot attached
<img width="1421" alt="screen shot 2017-09-11 at 3 15 55 pm" src="https://user-images.githubusercontent.com/2165463/30279103-295199da-9704-11e7-8f2b-a93db40aa077.png">

**Comment by mazameli** [comment]
Binning is actually a feature that will be shipped in version 0.26, but the frontend was accidentally included in 0.25.2 without the backend. 😑  Closing as a duplicate of #5653

**Comment by achanda** [comment]
Thanks for the reply. When is the 0.26 release scheduled?
